src/modules/builtin/cd.rs

```diff
@@ -0,0 +1,43 @@
+use heraclitus_compiler::prelude::*;
+use crate::modules::expression::expr::Expr;
+use crate::modules::condition::failed::Failed;
+use crate::docs::module::DocumentationModule;
+use crate::translate::module::TranslateModule;
+use crate::utils::{ParserMetadata, TranslateMetadata};
+
+#[derive(Debug, Clone)]
+pub struct Cd {
+    value: Expr,
+    failed: Failed
+}
+
+impl SyntaxModule<ParserMetadata> for Cd {
+    syntax_name!("ChangeDirectory");
+
+    fn new() -> Self {
+        Cd {
+            value: Expr::new(),
+            failed: Failed::new()
+        }
+    }
+
+    fn parse(&mut self, meta: &mut ParserMetadata) -> SyntaxResult {
+        token(meta, "cd")?;
+        syntax(meta, &mut self.value)?;
+        syntax(meta, &mut self.failed)?;
+        Ok(())
+    }
+}
+
+impl TranslateModule for Cd {
+    fn translate(&self, meta: &mut TranslateMetadata) -> String {
+        let value = self.value.translate(meta);
+        format!("cd {} || exit", value)
+    }
+}
+
+impl DocumentationModule for Cd {
+    fn document(&self, _meta: &ParserMetadata) -> String {
+        "Move the current working directory".to_string()
```

Comment:
Whenever user uses `cd` it will generate line `"Move the current working directory"` which doesn't tell us anything about the API.

```suggestion
        "".to_string()
```

The reason is that the change you proposed @Mte90 will transform this code:

```
fun foo() {

}

cd "path/to/file"
echo "Hello world"
cd "other/path/to/file"
```

To the following documentation:

````md
# `foo`
```ab
fun foo()
```
Move the current working directory
Move the current working directory
````

Reply:
Well we need to document all the modules otherwise doesn't make so much sense that code block.